project/profiles/capacity4more/modules/c4m/content/c4m_content_article/c4m_content_article.module

```diff
@@ -146,3 +146,35 @@ function c4m_content_article_c4m_offcanvas_trigger_label_left_alter(&$trigger_la
     $trigger_label['icon'] = 'filter';
   }
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function c4m_content_article_preprocess(&$variables) {
+  if ($variables['theme_hook_original'] != 'page') {
+    return;
+  }
+
+  if (empty($variables['node'])) {
+    return;
+  }
+
+  if ($variables['node']->type != 'article') {
+    return;
+  }
+
+  // Since the first element on the tree is actually the title we remove
+  // it because it should not have the arrow icon next to it (the <i> tag).
+  $vocab_geo = $variables['node']->c4m_vocab_geo[LANGUAGE_NONE];
+  $vocab_geo = array_slice($vocab_geo, 1);
+
+  // Go through the array and add an <i> tag, before the text, to each of them.
+  foreach ($vocab_geo as $geo) {
+    $geo_taxonomy = taxonomy_term_load($geo['tid']);
+
+    // Add an <i> tag to the 'Regions & Countries' tree children.
```

Comment:
Also, there is a space at the beginning of the string, is that on purpose ?
if no then remove it, if yes then explain why...